match/lib/match/utils.rb

```diff
@@ -58,6 +58,16 @@ def self.get_cert_info(cer_certificate_path)
       return {}
     end
 
+    def self.check_cert_validity(cer_certificate_path)
```

Comment:
I'm kind thinking that this method shouldn't have any `UI.error` or `UI.verbose` (or `UI.message` as it currently is implemented in it). Since this is a utils class, this method should be one that can be used in other parts of the code without influencing any UI. 

I think I would suggest renaming this to `is_cert_valid?` (as that is what this is checking **and then** doing then either do a `UI.user_error!` (since I think the goal of this PR was to exit out early) or a `UI.verbose` if cert is good. Thoughts on that?

Reply:
very useful comment, totally agree on this
i made some change to take your advice into account
+ i only check end date of validity because start date is - for apple certificate- necessary its creation date, so it is always in the past

Reply:
Using user_error makes the test "Match Match::Runner uses existing certificates and profiles if they exist" fail, since this certificate seems outdated. 
Is there a way to renew it?